### PR TITLE
[soltest] Fix hex string update

### PR DESCRIPTION
**Pull request by erak**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -43,6 +44,7 @@ Bugfixes:
 Build System:
  * Soltest: Add support for arrays in function signatures.
  * Soltest: Add support for struct arrays in function signatures.
+ * Soltest: Add support for left-aligned, unpadded hex string literals.
```
Comment (chriseth) [inline Changelog.md:49]: Is this in 0.5.6 on purpose?

**Review by chriseth — DISMISSED** [review]

File: Changelog.md (hunk 1 above)
Comment (erak) [reply Changelog.md:49]: Yes! That's the entry we merged into the 0.5.5 section by accident.

**Comment by chriseth** [comment]
Needs a rebase...

**Comment by erak** [comment]
Updated.

**Review by chriseth — APPROVED** [review]

**Pull request merged**